test/biz-route-test.js

```diff
@@ -29,77 +29,103 @@ const testbiz = {
 const url = `http://localhost:${process.env.PORT}/api/biz`;
 
 describe('Biz-router-test', function(){
- debug('bizRouter');
- before( done => {
-   var user = new User(testUser);
-   user.generatePasswordHash(testUser.password)
-   .then( () => user.save())
-   .then( () => user.generateToken())
-   .then( token => {
-     this.token = token;
-     this.userId = user._id;
-     done();
-   })
-   .catch(done);
- })
- after( done => {
-   User.remove({})
-   .then( () => done())
-   .catch(done);
- })
- after( done => {
-   Biz.remove({})
-   .then( () => done())
-   .catch(done);
- })
- describe('Biz:POST route', () => {
-   debug('Biz: POST route');
-   it('should test for invalid path', done => {
-    debug('valid body with invalid request');
-    request.post(`${url}/abcd`)
-    .send(testbiz)
-    .end( (err, res) => {
-      expect(res.status).to.equal(404);
-      console.log('******************res.status', res.status);
-      // expect(Error.msg).to.equal('Unauthorized');
+  debug('bizRouter');
+  before( done => {
+    var user = new User(testUser);
+    user.generatePasswordHash(testUser.password)
+    .then( () => user.save())
+    .then( () => user.generateToken())
+    .then( token => {
+      this.token = token;
+      this.userId = user._id;
       done();
+    })
+    .catch(done);
+  });
+  before( done => {
+    var newUser = new User({
+      username: 'FakeUser',
+      email: 'fakeUser@test.com',
+      password: 'fake'});
+    newUser.generatePasswordHash(testUser.password)
+    .then( () => newUser.save())
+    .then( () => newUser.generateToken())
+    .then( token => {
+      this.newToken = token;
+      User.findByIdAndRemove(newUser._id, function (err, user) {
+        user.remove();
+        done();
+      });
+    })
+    .catch(done);
+  });
+
+  after( done => {
+    User.remove({})
+    .then( () => done())
+    .catch(done);
+  });
+  after( done => {
+    Biz.remove({})
+    .then( () => done())
+    .catch(done);
+  });
+  describe('Biz:POST route', () => {
```

Comment:
A general 404 route test should maybe be in a separate test file. We don't need to test this in every file.